frontend/src/components/DeckyRouterState.tsx

```diff
@@ -10,12 +12,16 @@ export type RoutePatch = (route: RouteProps) => RouteProps;
 
 interface PublicDeckyRouterState {
   routes: Map<string, RouterEntry>;
-  routePatches: Map<string, Set<RoutePatch>>;
+  routePatches: Map<UIMode, Map<string, Set<RoutePatch>>>;
 }
 
 export class DeckyRouterState {
   private _routes = new Map<string, RouterEntry>();
-  private _routePatches = new Map<string, Set<RoutePatch>>();
+  // Update when support for new UIModes is added
```

Comment:
What other UIModes are there? I guess SteamVR might be seperate from BPM?

Reply:
I have no idea how vr works but I plan to look into it soon

it'll be interesting as it can run at the same time as the desktop ui